src/components/document/document-content.tsx

```diff
@@ -99,9 +101,11 @@ export class DocumentContentComponent extends BaseComponent<IProps, IState> {
   }
 
   public render() {
-    const {viaTeacherDashboard} = this.props;
+    const {viaTeacherDashboard, documentSelectedForComment} = this.props;
+    const documentClass = classNames("document-content", viaTeacherDashboard ? "document-content-smooth-scroll" : "",
+                                     documentSelectedForComment ? "comment-select" : "");
```

Comment:
Here's where `classNames()` is an improvement. Note that this would more conventionally be written using the object form rather than the ternaries:
```suggestion
    const documentClass = classNames("document-content", {
                                        "document-content-smooth-scroll" : viaTeacherDashboard,
                                        "comment-select": documentSelectedForComment });
```